content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/float/1. Float.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 1. Float
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Float.OI
+  - ib-19.3.Datatype-Primitief-Float.OI
+---
+
+## Wat is een Float?
+Het datatype `float` in C# is een numeriek datatype dat gebruikt wordt om decimale waarden op te slaan. Het is handig wanneer je een getal met een fractie wilt opslaan, zoals bijvoorbeeld bij meetgegevens of percentages. Een `float` biedt ongeveer 7 cijfers van precisie en gebruikt 32 bits voor opslag.
+
+Om een `float`-waarde in te voeren, gebruik je de letter `f` achter het getal, anders zal C# het getal standaard als een `double` interpreteren.
+
+> [!TIP] Casus  
+> Stel je ontwikkelt een applicatie voor een weerstation die real-time temperatuurmetingen bijhoudt. Deze meetgegevens worden elke minuut opgevraagd en opgeslagen in `float`-variabelen om het geheugengebruik te beperken en voldoende nauwkeurigheid te behouden. Door het gebruik van het `float`-type kan de applicatie efficiënt omgaan met grote hoeveelheden meetgegevens, terwijl de nauwkeurigheid van de temperatuurmetingen behouden blijft.
```

Comment:
Tweede zin loopt beetje krom en is erg lang

Reply:
Hoop dat het nu beter is